benchmark-service/benchmark/static/js/autocomplete.js

```diff
@@ -0,0 +1,142 @@
+function auto_complete(place_holder, str_selector) {
+     return new autoComplete({
+                   data: {                              // Data src [Array, Function, Async] | (REQUIRED)
+                     src: files,
+                     cache: true
+                   },
+                   sort: (a, b) => {                    // Sort rendered results ascendingly | (Optional)
+                       if (a.match < b.match) return -1;
+                       if (a.match > b.match) return 1;
+                       return 0;
+                   },
+                   placeHolder: place_holder,     // Place Holder text                 | (Optional)
+                   selector: str_selector,           // Input field selector              | (Optional)
+                   threshold: 2,                        // Min. Chars length to start Engine | (Optional)
+                   debounce: 300,                       // Post duration for engine to start | (Optional)
+                   searchEngine: "strict",              // Search Engine type/mode           | (Optional)
+                   resultsList: {                       // Rendered results list object      | (Optional)
```

Comment:
Let's remove the comments here. Generally, Hail team avoids in-line comments like this because they get out of date quickly. That's because we don't maintain them, of course, but the effort of maintaining them doesn't seem worth it. If we need to understand these parameters, we can always check the [API reference for autocomplete.js](https://tarekraafat.github.io/autoComplete.js/#/?id=api-configuration).